flink-libraries/flink-table/src/main/scala/org/apache/flink/table/codegen/generated.scala

```diff
@@ -35,7 +35,11 @@ case class GeneratedExpression(
     resultTerm: String,
     nullTerm: String,
     code: String,
-    resultType: TypeInformation[_])
+    resultType: TypeInformation[_]) {
+
+  var leftNullTerm: String = _
```

Comment:
I don't think we need these attributes.
We can check with `${codeGenerator.input1Term} == null` whether an input record is null.